js/components/root.js

```diff
@@ -25,13 +25,13 @@
 		'	<template v-if="missing.length">' +
 		'		<h3>' + t('caniupdate', 'Missing updates') + '</h3>' +
 		'		<ul>' +
-		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/apps/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
+		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
```

Comment:
you are breaking the URL

Reply:
That URL doesn't work for me. Logged in or not.

Reply:
All links on the appstore contain `apps/`:
https://apps.nextcloud.com/apps/contacts